src/west/app/project.py

```diff
@@ -844,6 +844,53 @@ def do_run(self, args, user_args):
                 failed.append(project)
         self._handle_failed(args, failed)
 
+        # List projects not tracked by manifest.
+        projects = [project.abspath for project in self._cloned_projects(args, only_active=True)]
+        topdir = Path(self.topdir)
+        untracked = []
+
+        self.do_find_untracked(topdir, projects, untracked)
+
+        ignored_config = self.config.get('status.ignore')
+        ignored = ignored_config.split(',') if ignored_config else []
+        # Ignore .west folder in untracked projects
+        ignored.append('.west/')
+        for ignore in ignored:
+            try:
+                untracked.remove(ignore)
+            except:
+                pass
+
+        if len(untracked) > 0:
+            self.banner('untracked:')
```

Comment:
```suggestion
            self.banner('Unknown:')
```
Please do not re-use the same presentation as git to avoid misleading users into thinking this line comes from git.